pinot-plugins/pinot-environment/pinot-azure/src/main/java/org/apache/pinot/plugin/provider/AzureEnvironmentProvider.java

```diff
@@ -40,21 +40,17 @@
 import org.apache.http.util.EntityUtils;
 import org.apache.pinot.spi.env.PinotConfiguration;
 import org.apache.pinot.spi.environmentprovider.PinotEnvironmentProvider;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 
 /**
  * Azure Environment Provider used to retrieve azure cloud specific instance configuration.
  */
 public class AzureEnvironmentProvider implements PinotEnvironmentProvider {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(AzureEnvironmentProvider.class);
-
   protected static final String MAX_RETRY = "maxRetry";
   protected static final String IMDS_ENDPOINT = "imdsEndpoint";
-  protected static final String CONNECTION_TIMEOUT = "connectionTimeout";
-  protected static final String REQUEST_TIMEOUT = "requestTimeout";
+  protected static final String CONNECTION_TIMEOUT_MILLIS = "connectionTimeoutMillis";
+  protected static final String REQUEST_TIMEOUT_MILLIS = "requestTimeoutMillis";
```

Comment:
To be consistent with other config keys in the code base
```suggestion
  protected static final String CONNECTION_TIMEOUT_MS = "connectionTimeoutMs";
  protected static final String REQUEST_TIMEOUT_MS = "requestTimeoutMs";
```

Reply:
@Jackie-Jiang  Saw some references where they are using MILLIS as suffix too like in StreamConfig.java.
Let me know if its fine to keep MILLIS as suffix.

Reply:
Agreed. We don't seem to have a convention either way, so I am fine having this as it is. @Jackie-Jiang let me know if we are moving towards a convention that I am not aware of.

Reply:
Seems we have both configs, so either way is fine to me